dandiapi/api/apps.py

```diff
@@ -50,6 +50,8 @@ def ready(self):
         import dandiapi.api.checks  # noqa: F401, RUF100
         import dandiapi.api.signals  # noqa: F401
 
+        logging.getLogger().setLevel(settings.DANDI_LOG_LEVEL)
```

Comment:
Thanks @waxlamp for catching the initial limitation! but I do not think we should go this wild, what about affecting only `dandi` ? i.e. smth like

```suggestion
        logging.getLogger(__name__.split('.', 1)[0]).setLevel(settings.DANDI_LOG_LEVEL)
```
?  otherwise, if to debug our code we turn to DEBUG level globally -- could lead to excessive amount of noise from all kinds of libraries we use.  WDYT?

Reply:
Ah I see what you were going for now. I've added this back (along with explanatory comment) in dc3549d.